db/fixtures/miq_product_features.yml

```diff
@@ -5,6 +5,21 @@
 :identifier: everything
 :children:
 
+# Container Deployment Create
+- :name: Container Deployment
+  :description: Container Deployment
+  :feature_type: node
+  :identifier: container_deployment_create
+  :children:
+  - :name: Container Deployment Show
```

Comment:
should the show and show_list be children of create here? it's likely I don't fully understand how it works :)